src/index.html

```diff
@@ -6,16 +6,30 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>JSHeroes Bootcamp</title>
     <link rel="stylesheet" href="style.css" />
+    <script src='script.js' defer></script>
```

Comment:
👍 Looks good, can open some discussions around how the browser renders HTML. Not to overload the mentees with details, but they can come back later to these examples and understand this things better.